### DISCUSSION TITLE
Place HBS files near JS files

### DISCUSSION THREAD

**Discussion by nightrider77** [handlebars-lang/handlebars.js#1938] [Q&A]
Hello.

My project is divided into 3 logical modules:

a) Admin area
b) User area
c) Agent area

Is it possible to make the following project structure:

**admin**
____ adminProfile.js
____ adminProfile.hbs
**user**
____ userAccount.js
____ userAccount.hbs
**agent**
____ agentControl
______ agentControlPage1.hbs
______ agentControlPage1.js
______ agentControlPage2.hbs
______ agentControlPage2.js
____ index.js
____ index.hbs

Thank you.

**Comment by jaylinski** [comment]
Yes, since handlerbars (this library) doesn't care about folder structure at all. It just converts a template (*.hbs) to HTML.
